src/Stratis.Bitcoin/P2P/Peer/NetworkPeerServer.cs

```diff
@@ -233,12 +241,12 @@ private bool AllowClientConnection(TcpClient tcpClient)
             if (endpointCanBeWhiteListed)
             {
                 this.logger.LogTrace("(-)[ENDPOINT_WHITELISTED_ALLOW_CONNECTION]:true");
-                return true;
+                return (true, "Endpoint Whitelisted, Allow Connection even if in IBD.");
             }
 
             this.logger.LogDebug("Node '{0}' is not white listed during initial block download.", clientLocalEndPoint);
 
-            return false;
+            return (false, "Node in IBD, Endpoint not whitelisted, connection refused.");
```

Comment:
Change the string to: `Inbound Refused: Endpoint not whitelisted during IBD.`

Reply:
same as before